core/public/company/company.controller.js

```diff
@@ -36,49 +36,45 @@ angular.module('Xtern')
         };
     }])
     .controller('CompanyRecruiting', ['$scope', '$state', 'ProfileService', 'CompanyService', function ($scope, $state, ProfileService, CompanyService) {
-        var self = this;
         $scope.recruitmentList = [];
 
         $scope.companyData = getToken("organization");
-            ProfileService.getStudentDataForIds($scope.companyData.studentIds, function(data) {
-                $scope.recruitmentList = data;
-            });
+        CompanyService.getOrganizationStudents(function(data) {
+            $scope.recruitmentList = data;
+        });
 
         $scope.sortableOptions = {
             containment: '#table-container',
             containerPositioning: 'relative'
         };
 
-        $scope.removeRecruit = function (_id) {
-            console.log("remove recruit:");
-            console.log(_id);
-            CompanyService.removeStudentFromWishList(_id, function(data) {
+        $scope.removeRecruit = function (key) {
+            console.log("remove recruit: "+key);
```

Comment:
We will. This is coming off a complete backend rewrite. The logs are there so we can make sure everything still triggers properly. 

We have plenty of console.log(s) that will get pulled before the app goes into prduction